connectivity/pxp_reporter_makefile/README.md

```diff
@@ -0,0 +1,89 @@
+SOURCEROOT is the path to the top level directory of the SDK (e.g. SDK_10.0.12.146). APP_SOURCE_DIR in the path to the top level project source files (e.g. <path_on_your_machine>/prox_reporter_makefile/).
+
+A separate build configuration is available for each of the build variants (Debug_QSPI, Debug_QSPI_SUOTA, Debug_RAM, Release_QSPI, Release_QSPI_SUOTA, Release_RAM). A user MUST specify a build configuration by setting the BUILD_CONFIG variable when calling make 
+in order to run the default target.
+
+Example call to build the Debug_QSPI build configuration:
+
+```
+make BUILD_CONFIG=debug_qspi
+```
+
+Example call to build Debug_QSPI build configuration with different SOURCEROOT and APP_SOURCE_DIR:
+
+```
+make BUILD_CONFIG=debug_qspi SOURCEROOT=<path_to_sdk>/SDK_10.0.12.146 APP_SOURCE_DIR=<path_to_examples_directory>/BLE_SDK10_examples/connectivity/pxp_reporter_makefile
+```
+
+Please note on Windows paths default to using a backslash (`\`) but the paths above should use a forward slash (`/`).
+
+A separate output directory is created for each build configuration: 
+
+![build_output_dirs](assets/build_output_dirs.png)
+
+For each build configuration, the top level output directory contains the primary build artifacts including the .map, .elf, and .bin files. The user may optionally change the names of these artifacts by specifying the the BUILD_ARTIFACT_NAME when building.
+The sdk and user directories contain the .o and .d outputs for each file built.
+
+![debug_qspi_output_top](assets/debug_qspi_output_top.png)
+
+To clean you have two options. To clean a specific build configuration output (e.g. Debug_QSPI) you must specify the BUILD_CONFIG when calling clean:
+
+```
+make clean BUILD_CONFIG=debug_qspi
+```
+
+Please note if your build requires redefining the SOURCEROOT and APP_SOURCE_DIR variables, these variables must also be defined when cleaning:
+
+```
+make clean BUILD_CONFIG=debug_qspi SOURCEROOT=<path_to_sdk>/SDK_10.0.12.146 APP_SOURCE_DIR=<path_to_examples_directory>/BLE_SDK10_examples/connectivity/pxp_reporter_makefile
+```
+
+If no BUILD_CONFIG is specified when calling clean, all build configurations outputs will be cleaned. 
+
+For QSPI build configurations (Debug_QSPI, Debug_QSPI_SUOTA, Release_QSPI, Release_QSPI_SUOTA) an additional post build step is performed to create a .img file with the necessary product headers and image header as described in the [DA1469x Software Platform Reference](http://lpccs-docs.renesas.com/um-b-092-da1469x_software_platform_reference/User_guides/User_guides.html#flash-layout). 
+
+### Verify
+
+A diff tool, such as [WinMerge](https://winmerge.org/?lang=en) on Windows, can be used to compare the makefile generated .bin to that produced by the pxp_reporter Eclipse project to verifiy they are identical. 
```

Comment:
In my case, the binaries were not the same. I'm not sure if this will always be the same, as it depends on build/link order. So I would recommend to remove this section.

Reply:
Agreed it depends on build/link order. Not sure why the order would change on your machine as every directory should be searched in the same order as defined by SDK_SOURCE_SUBDIRS. 

For now I have removed this section in https://github.com/dialog-semiconductor/BLE_SDK10_examples/pull/30/commits/1ba3fc9295fb8640ee6ef1eb723ecbc2f42c2ad1. I'll try to revisit at a later date.